hack/gen-resources/generators/cluster_generator.go

```diff
@@ -193,9 +195,23 @@ func (cg *ClusterGenerator) Generate(opts *util.GenerateOpts) error {
 
 		log.Print("Get cluster credentials")
 		caData, cert, key, err := cg.getClusterCredentials(namespace, releaseSuffix)
+
+		for o := 0; o < 5; o++ {
+			if err != nil {
+				if o > 0 {
```

Comment:
I don't think we need the `if`... if we hit this line, we already know there was a failure, right?

Reply:
@crenshaw-dev ah yes, you're right. I had that in there from a previous revision that I abandoned.